### PR TITLE
replacing R with r in ls challenge

### PR DESCRIPTION
**Pull request by sstevens2**
First draft to change `-R` to `-r` in challenge.  Based on issue #892.

Might need some edits but a first pass at it.

### CODE REVIEW

File: _episodes/02-filedir.md [hunk 1]
```diff
@@ -421,17 +421,19 @@ To **quit** the `man` pages, press <kbd>Q</kbd>.
 > {: .solution}
 {: .challenge}
 
-> ## Listing Recursively and By Time
+> ## Listing in Reverse and By Time
```
[the remark this answers is not in the document]
Comment (gdevenyi) [reply _episodes/02-filedir.md]: Or..
> Chronological reverse order

[the remark this answers is not in the document]
Comment (gcapes) [reply _episodes/02-filedir.md]: Thanks - Let's go for:
> Listing in reverse chronological order

**Comment by gcapes** [comment]
Great, thanks!

**Comment by gcapes** [comment]
Fix #892

**Pull request merged**